docs/tutorials/google/colab.ipynb

```diff
@@ -159,7 +159,7 @@
    "source": [
     "## Authenticate and install Cirq\n",
     "\n",
-    "For details of authentication and installation, please see [go/quantum-engine-quickstart](http://go/quantum-engine-quickstart). \n",
+    "For details of authentication and installation, please see [Get started with Quantum Computing Service](start.ipynb).\n",
```

Comment:
Broken link? (May need to be `./start.ipynb`)

Reply:
IIUC, this should work once it's up on the website. 

See https://github.com/quantumlib/Cirq/blob/40bc85651a481f5809e2d6772233c623cf690591/docs/tutorials/google/start.ipynb#L608 for a similar example.